src/app/components/luzmo-dashboard.tsx

```diff
@@ -19,21 +19,23 @@ interface Dashboard {
 }
 
 export default function LuzmoDashboard({ authKey, authToken, dashboards }: Props) {
+  // Check if the dashboards array is not empty
+  if (dashboards.length === 0) {
+    return null; 
+  }
+
+  // Get the first dashboard
+  const firstDashboard = dashboards[0];
+  const { id } = firstDashboard;
+
   return (
-    <>
-      {dashboards.map((dashboard: Dashboard) => {
-        const { id } = dashboard;
-        return (
-          <LuzmoDashboardComponent
-            key={id}
```

Comment:
We probably don't need to provide a key now since this is now only returning a single instance of the component.

Reply:
Yes, that makes sense